### PR TITLE
Specify ALPN tokens

### PR DESCRIPTION
**Pull request by pthatcherg**

### CODE REVIEW

File: cs.html [hunk 1]
```diff
@@ -672,19 +672,23 @@ <h2>Constructors</h2>
...
+                and value of <var>serializedOrigin</var>.
                 -->
                 <li>Establish a QUIC connection to the address identified by the
-                <var>parsedUrl</var>'s host and port.  The user agent MUST
-                include a QUIC transport parameter with ID of web_origin(0x330) 
-                <!-- TODO: Register the transport parameter with IANA -->
-                and value of <var>serializedOrigin</var>.</li>
+                <var>parsedUrl</var>'s host and port.  
+                During connection establishment, use of this API must be indicated 
+                by selecting the ALPN token "wq" in the crypto handshake.
```
Comment (shampson) [inline cs.html]: Should we reference the rfc?
https://tools.ietf.org/html/rfc7301

**Comment by shampson** [comment]
Shouldn't the ALPN also be used in the P2P case as part of its connection establishment?

**Comment by pthatcherg** [comment]
Yes, I'll add a reference to https://tools.ietf.org/html/rfc7301.  

Didn't I cover the p2p case in index.html?

File: cs.html (hunk 1 above)
Comment (pthatcherg) [reply cs.html]: Added

**Pull request merged**